rpc/client/rpc_test.go

```diff
@@ -576,10 +577,10 @@ func TestClientMethodCalls(t *testing.T) {
 func getMempool(t *testing.T, srv service.Service) mempool.Mempool {
 	t.Helper()
 	n, ok := srv.(interface {
-		Mempool() mempool.Mempool
+		RPCEnvironment() *rpccore.Environment
```

Comment:
Does this imply we could get rid of the `Mempool()` method too?

Reply:
I think I did?

Reply:
yup, it's gone!

Reply:
Oh, oops, I missed it. Yay! 🎉